index.html

```diff
@@ -601,6 +601,180 @@ <h2 class="sg-section-title structure">Forms</h2>
                         </header>
                         <div class="sg-section-content structure">
                             
+
+    
+        <h3 class="section-sub-title">Example Form</h3>
+
+        <form action="" method="post">
+            <div class="error-block form-error">
+                <p class="error">This is an example form error</p>
+            </div>
+
+            
+                <fieldset class="text-fieldset">
+                    <div class="control-group text required">
+                        <label for="field-name" class="control-label">
+                            Text input
+                            <span class="asterisk">*</span>
+                        </label>
+                        <div class="controls">
+                            <input class="text-input" id="field-name" type="text" name="name" placeholder="Text input" aria-required="true">
+                        </div>
+                    </div>
+
+                    <div class="control-group email required">
+                        <label for="field-email" class="control-label">
+                            Email input
+                            <span class="asterisk">*</span>
+                        </label>
+                        <div class="controls">
+                            <input class="email-input" id="field-email" type="email" name="email" placeholder="Email input" aria-required="true">
```

Comment:
Hmm, good point. I guess I am just so used to seeing them always output the same way.

Another question. Should we be using `id` at all? We try to avoid it nowadays for most elements, as `id` should be unique within the document, and generally classes make more sense for styling. 

I'd argue we probably ought to remove it for styling purposes. What is the page has two forms, both with `name` fields - the IDs will be duplicated and of no help. Should specific field identifiers be at the `.control-group` level? e.g `<div class="control-group field-name-email">` or something? Then you can style other parts of that field (errors and label) differently for specific fields, and you can target individual fields by form class name and `control-group` class?

Is there any case where we specifically need `id` attributes for javascript? I can't think of one really, as we ought to be using `name` for form related activities.